models/template/work/threed_input.nml

```diff
@@ -147,27 +145,39 @@
    /
 
 &obs_sequence_tool_nml
-   filename_seq      = 'obs_seq.one', 'obs_seq.two',
-   filename_out      = 'obs_seq.processed',
-   first_obs_days    = -1,
-   first_obs_seconds = -1,
-   last_obs_days     = -1,
-   last_obs_seconds  = -1,
-   print_only        = .false.,
-   gregorian_cal     = .false.
```

Comment:
Has gregorian_cal been removed from the &obs_sequence_tool_nml in general? Or is it just optional and not needed here?